operations/README.md

```diff
@@ -41,6 +41,8 @@ This section describes all operations supported by the API, organised here by th
 | [Update customer](customers.md#update-customer) | Updates personal information of a customer |
 | [~~Merge customers~~](customers.md#merge-customers) | **Deprecated!** Please use [Merge accounts](accounts.md#merge-accounts) instead. |
 | [Add customer file](customers.md#add-customer-file) | Attaches the specified file to the customer profile |
+| [Delete identity documents](identitydocuments.md#delete-identity-documents) | Deletes selected identity documents |
+| [Clear identity documents](identitydocuments.md#delete-identity-documents) | Deletes all identity documents for selected customers |
```

Comment:
issue: Link
```suggestion
| [Clear identity documents](identitydocuments.md#clear-identity-documents) | Deletes all identity documents for selected customers |
```

Reply:
Good catch, the file didn't saved and thus didn't put into commints.